File: codes/1603-running-sum-of-1d-array/running-sum-of-1d-array.cpp
#include <vector>
using namespace std;
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int sum=0;
        vector <int> num;
        for (int i: nums){
            sum+=i;
            num.push_back(sum);
       
        }
        return num;
    }
};

File: codes/1603-running-sum-of-1d-array/running-sum-of-1d-array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        num=[]
        sum=0
        for i in nums:
            sum+=i
            num.append(sum)
        return num


        # sum=0
        # num=[]
        # for i in range(0,len(nums)):
        #     for j in range(i):
        #         sum=i+j
        #         num.append(sum)
        #         return num
